addons/note/static/src/js/note.js

```diff
@@ -0,0 +1,9 @@
+openerp.note = function(instance) {
+    _t = instance.web._t;
+    instance.web_kanban.KanbanGroup.include({
+        init: function() {
+            this._super.apply(this, arguments);
+            this.title = ((this.dataset.model === 'note.note') && (this.title === _t('Undefined'))) ? _t('Shared') : this.title;
+        },
+    });
+};
```

Comment:
This will need a rebase and be fixed with the new JavaScript dependencies management